cmd/gb-vendor/purge.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"fmt"
+	"go/parser"
+	"go/token"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/constabulary/gb"
+	"github.com/constabulary/gb/cmd"
+	"github.com/constabulary/gb/cmd/gb-vendor/vendor"
+)
+
+func init() {
+	registerCommand(PurgeCmd)
+}
+
+func parseImports(root string) (map[string]struct{}, error) {
+	var found = struct{}{}            // Does not take any space
+	pkgs := make(map[string]struct{}) // Set
```

Comment:
Map values always take at least one byte for the key, so map[string]bool makes a better set.

You also get to do 

if m["thing"] { // found }

rather than

if _, found := m["thing"]; found { ... }